src/io/files.js

```diff
@@ -13,6 +13,7 @@ define(function (require) {
 
   var p5 = require('core');
   var reqwest = require('reqwest');
+  require('helpers');
```

Comment:
If #720 lands before this does, this will need to be changed to 'core/error_helpers'. If this lands before #720 does, that PR can be rebased on top of this one to account for the change.